{{cookiecutter.plugin_name}}/src/{{cookiecutter.module_name}}/_writer.py

```diff
@@ -7,15 +7,18 @@
 Replace code below according to your needs.
 """
 from __future__ import annotations
-from typing import List
-from numpy.typing import ArrayLike
+from typing import TYPE_CHECKING, List, Any, Sequence, Tuple, Union
 
+if TYPE_CHECKING:
+    DataType = Union[Any, Sequence[Any]]
```

Comment:
`DataType` is a bit excessive maybe. Just replace w `Any`?

Reply:
can do, i do think broader is better.
but fwiw, that _is_ basically the layerdata protocol we expect.  (structural type as opposed to literal type).  so, without saying `Union[numpy.ndarray, dask.Array, etc...]` it beings captures what a user can expect from that object

Reply:
oh, sorry, my brain was still in npe2 😂  ... where we have another thing called DataType don't listen to me :)